build.gradle

```diff
@@ -60,8 +60,6 @@ allprojects {
   if (it.path != ':integrations:spark:spark-3.5:openhouse-spark-3.5-itest') {
     configurations.all {
       resolutionStrategy {
-        force 'com.fasterxml.jackson:jackson-bom:2.13.4'
-        force 'com.fasterxml.jackson.core:jackson-databind:2.13.4'
```

Comment:
are these changes relevant ?

Reply:
this may actually be dangerous to remove, and I will fix it, since idk what is currently relying on this.

but this strict version constraint breaks :apps:spark-3.5 because that also needs jackson 2.15